internal/config/properties.go

```diff
@@ -12,6 +12,7 @@ const (
 	iamPolicyS3Restricted = "iam.policy.s3.restricted.resource"
 	awsAccountId          = "aws.accountId"
 	awsMasterRole         = "aws.MasterRole"
+	managedPolicies       = "iam.managed.policies"
```

Comment:
config map property name and this field is not matching

Reply:
It was supposed to be "iam.managed.policies", I will change it in config map.